scripts/dss_v2_apdapter.py

```diff
@@ -0,0 +1,316 @@
+                if entity_type.endswith('_file'):
+                    # Fetch file contents, modify, and upload to bucket
+                    file_contents = json.loads(self.src_bucket.blob(f'blobs/{blob_key}').download_as_string())
+                    try:
+                        data_file_old_name = file_contents['file_core']['file_name']
+                    except KeyError:
+                        self.log_error(bundle_uuid, manifest_entry['name'], f"'file_core.file_name' not found in file")
+                        break
+                    try:
+                        data_file_new_name = data_file_new_names[data_file_old_name]
+                    except KeyError:
+                        self.log_error(bundle_uuid, manifest_entry['name'], f"Unknown 'file_core.file_name' value")
+                        break
+                    file_contents['file_core']['file_name'] = data_file_new_name
+                    result = self.upload_file_contents(file_contents, new_name, manifest_entry['content-type'])
+                    benchmarks.append((time.perf_counter() - t1, f"to upload {manifest_entry['name']}"))
+
+                else:
+                    result = self.copy_file(blob_key, new_name)
+                    benchmarks.append((time.perf_counter() - t1, f"to copy {manifest_entry['name']}"))
+
+            if not result:
+                self.log_error(bundle_uuid, manifest_entry['name'], f"Failed to copy file to {new_name}")
+
+        total_duration = 0
+        for duration, message in benchmarks:
+            total_duration += duration
+            log.debug(f'{round(duration, 4)} {message}')
+        log.debug(f'{round(total_duration, 4)} Total')
+
+    def upload_file_contents(self, file_contents: Mapping[str, Any], new_name: str, content_type: str) -> Blob:
+        """
+        Perform an upload of a dict (json file content) to a bucket
+        """
+        log.info('Uploading to: %s', new_name)
+        self.dst_bucket.blob(new_name).upload_from_string(data=json.dumps(file_contents, indent=4),
+                                                          content_type=content_type)
+        return self.dst_bucket.blob(new_name).exists()
+
+    def copy_file(self, blob_key: str, new_name: str) -> bool:
+        """
+        Perform a bucket to bucket copy
+        """
+        log.info('Copying to: %s', new_name)
+        src_blob = self.src_bucket.blob(f'blobs/{blob_key}')
+        dst_blob = self.src_bucket.copy_blob(blob=src_blob,
+                                             destination_bucket=self.dst_bucket,
+                                             new_name=new_name)
+        return isinstance(dst_blob, Blob)
+
+    def log_error(self, bundle_uuid: str, file_name: str, msg: str):
+        """
+        Log an error message and save a copy for re-reporting at end of script
+        """
+        self.errors.append((bundle_uuid, file_name, msg))
+        log.error(msg)
+
+    @classmethod
```

Comment:
I believe we've had this discussion many times before. There is NO point in declaring every method that doesn't use `self` as classmethod. Then your just typing extra text `@classmethod` and replacing an unused `self` argument with an unused `cls` argument. A method should be declared `@classmethod` only if the call site doesn't actually have an instance available. This is typically only true for methods that CREATE an instance, so called factory methods. Every other case of classmethod is questionable and deserves extra scrutiny.

This is especially annoying here because this class is a singleton so there isn't ANY substantial difference between a class/static method and an instance method.